tests/dashbio_demos/app_molecule3d.py

```diff
@@ -32,7 +32,8 @@ def header_colors():
 
 def description():
     return 'Molecule visualization in 3D - perfect for viewing ' \
-           'biomolecules like proteins, DNA and RNA'
+           'biomolecules like proteins, DNA and RNA. Includes ' \

```

Comment:
```suggestion
           'biomolecules such as proteins, DNA and RNA. Includes ' \
```
The casual style might be deliberate, but let's sound like scientists...